tests/foreman/ui/test_myaccount.py

```diff
@@ -199,19 +175,11 @@ def test_positive_update_password(self):
         """
         for password in _valid_string_data(max_len=254):
             with self.subTest(password):
-                with self.logged_test_user():
-                    self.my_account.navigate_to_entity()
-                    self.my_account.wait_until_element(
-                        locators['users.password'])
-                    self.my_account.assign_value(
-                        locators['users.password'], password)
-                    self.my_account.assign_value(
-                        locators['users.password_confirmation'], password)
-                    self.my_account.click(common_locators['submit'])
-                    self.my_account.wait_until_element_is_not_visible(
-                        common_locators["notif.success"])
-                    self.login.logout()
-                    self.login.login(self.account_user.login, password)
+                with self.session_test_user() as session:
```

Comment:
Think this is a destructive test, if the password is changed and the user fail to login for any reason all the tests will be affected as the password is not restored, think we should create a new user for this test, what do you think?